### PR TITLE
Yaml Reference Section

### PR DESCRIPTION
**Pull request by CamrynCarter**
Adding Yaml Runner Reference section to the documentation.

### CODE REVIEW

File: content/docs/yaml-reference/story-runner-components/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+| **action**         | Required for every operation, the action specifies what function will be performed.                                                                                                                                                                                                                             |
+| **description**    | Optional string that describes what the step is doing. For example, "Create the Asset My First Container".                                                                                                                                                                                                      |
+| **asset_label**    | For a series of steps run as one file, the asset label could be a friendly name used by later steps to refer back to an asset created in a previous step. If the asset already exists, this field may be used to reference the Asset ID in the form assets/add30235-1424-4fda-840a-d5ef82c4c96fa.               |
```
Comment (kjc-jitsuin) [inline content/docs/yaml-reference/story-runner-components/index.md:26]: Can we make it more clear that the assets/_asset ID_ is sourced from the existing asset? 

"assets/[_your_asset_ID_]" or similar, with the example?

Comment (CamrynCarter) [reply content/docs/yaml-reference/story-runner-components/index.md]: Yes, I updated all the example IDs throughout the YAML Reference to match the examples from our other sections - "assets/<asset-id>".

File: content/docs/yaml-reference/assets/index.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      - RecordEvidence
+    attributes:
+      arc_display_name: ev pump 1
+      arc_display_type: pump
+      arc_namespace: wipp
+      ev_pump: "true"
+    confirm: true
+```
+
+The output of this action would be:
```
Comment (kjc-jitsuin) [inline content/docs/yaml-reference/assets/index.md:50]: We should explain that this resulting output is visible in the UI

Comment (CamrynCarter) [reply content/docs/yaml-reference/assets/index.md]: added note with hyperlink to the UI

**Review by kjc-jitsuin — APPROVED** [review]

**Pull request merged**